examples/rl_gsm8k/utils.py

```diff
@@ -381,6 +397,20 @@ def launch_training(
                 "--deepspeed_config_file",
                 "conf/accelerate/deepspeed_stage3_bf16.json",
             ]
+            if is_multinode:
+                base_cmd.extend([
+                    "--num_machines",
+                    WORLD_SIZE,
+                    "--machine_rank",
+                    GLOBAL_RANK,
+                    "--main_process_ip",
+                    MASTER_ADDRESS,
+                    "--main_process_port",
+                    MASTER_PORT,
+                    "--deepspeed_multinode_launcher",
+                    "standard",
+                    "--same_network",
+                ])
```

Comment:
Would training multi-node training without accelerate work without DeepSpeed? If not, should add an exception.